parts/language/lang_vi.lua

```diff
@@ -515,7 +515,7 @@ return {
             moveFX="Move FX",
             clearFX="Clear FX",
             splashFX="Splash FX",
-            shakeFX="Field Sway",
+            shakeFX="Độ Rung Bảng",
```

Comment:
A small suggestion: translating Field Sway into "Độ nảy bảng" or "Độ lắc bảng"? Sound better (or not)


![image](https://user-images.githubusercontent.com/106439598/222905533-d4ca5caf-fa92-4f23-8351-f0a5a4abc30f.png)